TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
Not sure if that's a big issue, since this kickstarter package is part of the Neos base distribution.
I guess 99% of all Neos projects utilize `TYPO3.Neos.NodeTypes:Page` in some way and therefore the `TYPO3.Neos.NodeTypes` package. The 1% that does not can still adapt that line to their own needs when the start a new project.